src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -130,6 +131,7 @@ void TranslateSession::translate_graph(const ov::frontend::InputModel::Ptr& inpu
     for (const auto& operation_place : operation_places) {
         auto operation_decoder = operation_place->get_decoder();
         auto operation_name = operation_place->get_names()[0];
+        op_type_by_name[operation_name] = operation_decoder->get_op_type();
```

Comment:
Why only first item is using? Why dont you use "for(name : get_names) { ... }" ?

If it is a limitation - could you provide a description

Reply:
operation has only one name. You mix up with tensor names. No limitation is here